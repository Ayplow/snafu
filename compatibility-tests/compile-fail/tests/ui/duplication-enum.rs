use snafu::Snafu;

#[derive(Debug, Snafu)]
#[snafu(visibility(pub))]
#[snafu(visibility(pub))]
enum EnumError {
    AVariant,
}

fn main() {}
