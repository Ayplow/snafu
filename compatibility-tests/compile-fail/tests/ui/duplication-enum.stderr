error: Multiple `visibility` attributes are not supported on an error enum
 --> $DIR/duplication-enum.rs:5:9
  |
5 | #[snafu(visibility(pub))]
  |         ^^^^^^^^^^^^^^^
